### PR TITLE
Update nodejs.yml

### PR DESCRIPTION
**Pull request by ralfhandl**
Auto-refresh *.md and *.json files on push.

With this change it is no longer necessary to locally `npm run build` to refresh the derived files, this happens on push to a branch, including `master` when a pull request is merged.

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -21,6 +23,10 @@ jobs:
       run: |
         npm install
         npm run build --if-present
-        git diff --exit-code
+        git config user.name ${GITHUB_ACTOR}
+        git config user.email ${GITHUB_ACTOR}
```
Comment (HeikoTheissen) [inline .github/workflows/nodejs.yml:27]: Can't you give the email address here?

Comment (ralfhandl) [reply .github/workflows/nodejs.yml:27]: I first have to figure out how to get at it. Which I will, and we can use it as is for now.

Comment (ralfhandl) [reply .github/workflows/nodejs.yml:27]: And as I have a "how to create a vocab PR" session with Renzo in an hour, I'd rather have this merged before that.

Comment (ralfhandl) [reply .github/workflows/nodejs.yml]: Fixed

**Pull request merged**